obspy/CONTRIBUTORS.txt

```diff
@@ -9,6 +9,7 @@ Felix Bernauer
 Moritz Beyreuther
 Sébastien Bonaimé
 Lloyd Carothers
+Calum Chamberlain
```

Comment:
@megies should I rebase this now then?  I have addressed the comments below now too (when I push I will put some update highlights).